modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -880,3 +880,54 @@
 	new /obj/item/ammo_box/magazine/smartgun(src)
 	new /obj/item/ammo_box/magazine/smartgun(src)
 	new /obj/item/ammo_box/magazine/smartgun(src)
+
+//////////////////////////////////////////////////////////
+/////////////////KRAUT SPACE MAGIC G11////////////////////
+//////////////////////////////////////////////////////////
+
+/obj/item/gun/ballistic/automatic/g11
+	name = "\improper Armadyne G11 K2"
```

Comment:
```suggestion
	name = "\improper Armadyne G11 K2 Mk. VI"
```
NO MOM IT'S NOT THE 1980 GUN IT'S A REVISED VERSION FROM 500 YEARS FORWARD!!!

Reply:
Why don't we just pretend it's a new gun just designed by arnafuck